wunderlist-get-comments.html

```diff
@@ -0,0 +1,48 @@
+<script type="text/javascript">
+  RED.nodes.registerType('wunderlist-get-comments',{
+    category: 'function',
+    color: '#E84228',
+    defaults: {
+      config: { type: 'wunderlist-config', required: true },
+      listId: { value: '' },
+      taskId: { value: ''},
+      name: { value: '' }
+    },
+    inputs:1,
+    outputs:1,
+    icon: 'wunderlist-logo.png',
+    label: function() {
+      return this.name || 'get Wunderlist comments';
+    }
+  });
+</script>
+
+<script type="text/x-red" data-template-name="wunderlist-get-comments">
+  <div class="form-row">
+    <label for="node-input-config"><i class="fa fa-code"></i> Configuration</label>
+    <input type="text" id="node-input-config" placeholder="Configuration">
+  </div>
+  <div class="form-row">
```

Comment:
you mention that `taskId` will take precedence over `listId`
so I think it would make more sense to switch the order of the displayed fields here so the `taskId` field will be first and the `listId` field will be second

Reply:
Adjusted